starter-code/index.html

```diff
@@ -1,42 +1,102 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <title>Spotify</title>
+	<meta charset="UTF-8">
+	<title>Spotify</title>
+	<link rel="stylesheet" href="css/main.css">
 </head>
 <body>
-Premium Discover Help Download
-
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
-
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
-
-HD Music
-Listen to music as if you
-were listening live
-
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
-
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
+
+	<header>
+		<div class="logo">
+			<img src="images/spotify-logo.png" width="20%">
+		</div>
+		<nav>
+			Premium Discover Help Download
+		</nav>
+	</header>
+	<div class="clearfix"></div>
+
+	<div class="hero">
+		<div class="hero-text">
+			<!-- <center> -->
+			<h1>Music for everyone.</h1>
+			Spotify is now free on mobile, tablet and computer.<br>
+			Listen to the right music, wherever you are.
+			<!-- </center> -->
+		</div>
+	</div>
+
+	<div class="main-body">
+		
+		<div class="main-before-insert">
+
+			<div class="main-whats-on-spotify">
+				What’s on Spotify?
+			</div>
+			
+			<div class="feature-containers">
+
+				<div class="main-feature-1 float-l">
+					<img src="images/music-icon.png" width="10.5%">
+					<span class="spotify-green">Millions of Songs</span>
+					There are millions of
+					songs on Spotify
+				</div>
+
+				<div class="main-feature-2">
+					<img src="images/high-quality-icon.png" width="10.5%">
+					<span class="spotify-green">HD Music</span>
+					Listen to music as if you
+					were listening live
+				</div>
+
+				<div class="main-feature-3 float-r">	
+					<img src="images/devices-icon.png" width="10.5%">	
+					<span class="spotify-green">Stream Everywhere</span>
+					Stream music on your
+					smartphone, tablet or
+					computer
+				</div>
+
+			</div>
+		</div>
+
+		<div class="clearfix"></div>
+
+		<div class="main-body-insert">
+			<div class="left-text">
```

Comment:
Maybe it would be more practical if you divided this section into two divs (on that holds everything that's on the left side and the other one that holds everything that's on the right side) and after that just use the classes to position them.